IBPSA/Fluid/FixedResistances/CheckValve.mo

```diff
@@ -0,0 +1,120 @@
+within IBPSA.Fluid.FixedResistances;
+model CheckValve "Check valve that avoids flow reversal"
+  extends IBPSA.Fluid.BaseClasses.PartialResistance(
+    dp(nominal=6000),
+    final dp_nominal=dpValve_nominal + dpFixed_nominal,
+    final m_flow_turbulent=deltaM*abs(m_flow_nominal),
+    final from_dp=true,
+    final linearized=false,
+    allowFlowReversal=false);
```

Comment:
```suggestion
    allowFlowReversal=true);
```

allowsFlowReversal should be `true` since the flow does become slightly negative.